Michaelw/delete index (#7800)
DataDog/documentation: content/en/logs/indexes/_index.md

**Note**: Indexes daily quotas reset automatically at 2:00pm UTC (4:00pm CET, 10:00am EDT, 7:00am PDT).

## Data and index deletion

You can request deletion of data within a time frame, or an entire index, by clicking on the deletion icon next to an index name.

{{< img src="logs/indexes/delete_button.png" alt="index details"  style="width:70%;">}}

You will be presented with a modal containing two options:

  - Delete data within a specific time frame
  - Delete an entire index, along with all of its contained data

{{< img src="logs/indexes/delete_modal.png" alt="index details"  style="width:70%;">}}

> When you request the deletion of data, a support ticket will be automatically be generated on your behalf to support@datadoghq.com. Someone

Reviewer: ```suggestion
When you request the deletion of data, a support ticket is automatically generated on your behalf to support@datadoghq.com. Someone
```
